clvm/core_ops.py

```diff
@@ -47,6 +47,8 @@ def op_raise(args):
 
 
 def op_eq(args):
+    if args.list_len() != 2:
```

Comment:
I thought that we wanted equal to take unlimited arguments so you could assert equality between multiple things without needing multiple checks?

Reply:
perhaps. that's not what it does now though. If we make that change, we can change this check too.

Reply:
I'm not sold on that being desirable though, at least not in the low-level language. There's consistency with `>` and `>s` right now (in being binary operators).

Reply:
Maybe I'm paranoid, but I am also afraid that `=` with multiple values could lead to subtle consensus bugs across implementations.

Reply:
I agree, better to be conservative and only allow two arguments at the lowest level.